app/views/pages/primary-early-careers.html.erb

```diff
@@ -0,0 +1,83 @@
+<% meta_tag :title, 'Primary early career and trainee teachers' %>
+<% meta_tag :description, 'We support early career and trainee teachers to develop their teaching of primary computing through professional development, resources and local communities.' %>
+
+<%= render 'components/hero', hero_title: 'Primary early career and trainee teachers', subtitle: 'We support early career teachers and trainees to develop their computing subject knowledge and pedagogy through professional development, resources and local communities.', colour: 'jaffa-bg', full_width: true %>
```

Comment:
@maddyjoe the hero title and meta titles are slightly different:

Meta: Primary early career teachers and trainees
Hero: Primary early career and trainee teachers

Can you confirm if that's expected, or if they should match?

Reply:
apologies they should match and be as per figma hero (aka the meta needs changing to use Primary early career and trainee teachers). Nikki asked for this and I updated figma but forgot I'd put the meta on the ticket too.

Reply:
all resolved thank you